A-set/667A. Pouring Rain.py

```diff
@@ -1,9 +1,14 @@
+# from dust i have come, dust i will be
+
 import math
-d,h,v,e = map(int, input().split())
-drink_s = (4*v)/(math.pi * (d**2))
+
+d, h, v, e = map(int, input().split())
+
+drink_s = (4 * v)/(math.pi * (d**2))

```

Comment:
`(4 * v) / (math.pi * (d**2))` - there should be spaces before and after the `/`